k8s-ci/utils.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Sets additional required environment variables that aren't available in the
+# secrets.yml file, and performs other preparatory steps
+function prepareTestEnvironment() {
+  # Prepare Docker images
+  docker build --tag "gke-utils:latest" \
```

Comment:
I think `UNIQUE_TEST_ID` or similar would be useful here to prevent collisions between builds.

Reply:
`gke-utils:latest` should actually be a globally accessible image. it generally shouldn't change. i'll get rid o the docker build arg since it's not necessary